src/join/composer.jl

```diff
@@ -45,8 +46,8 @@ struct DataFrameJoiner
                                     "Symbol or Pair{Symbol, Symbol}."))
             end
         end
-        dfl_on = dfl[!, left_on]
-        dfr_on = dfr[!, right_on]
+        dfl_on = select(dfl, left_on, copycols=true)
+        dfr_on = select(dfr, right_on, copycols=true)
```

Comment:
```suggestion
        dfl_on = select(dfl, left_on, copycols=false)
        dfr_on = select(dfr, right_on, copycols=false)
```

Reply:
This is normal (really) and that is why review process is so important.

When I see comments by @nalimilan of my code I often ask myself how I could have written and committed something.

Reply:
And that is why I decided to involve in open source project at least a bit!